miden/tests/integration/cli/cli_test.rs

```diff
@@ -43,6 +43,7 @@ fn cli_run() -> Result<(), Box<dyn std::error::Error>> {
 }
 
 #[test]
+#[ignore] // until the compiler supports asserts with error messages issue #1705
```

Comment:
Issue #1705 is the issue which this PR will close - however I'm reading this comment as "issue 1705 tracks the compiler supporting asserts with error messages".

With no documentation as to what `is_prime.masp` does, I would instead remove these tests and open an issue in the VM repo to reinstate these tests so that we can track it.

Reply:
@paracetamolo you removed the tests right? In this case can you also remove the `is_prime.{masp, inputs}` files as well? And then open an issue to re-add a test that checks that the CLI can properly run `.masp` files (as we effectively undid some of the work done in #1683).